compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -643,35 +641,26 @@ class ComposeScene internal constructor(
     }
 
     private fun processRelease(event: PointerInputEvent) {
-        fun isOutsideFocusedOwner(): Boolean {
-            if (pressOwner != null) {
-                // The gesture started not outside of owner
-                return false
-            }
-            if (event.isAnyPointerDown) {
-                // The last pointer was not released yet
-                return false
-            }
-
-            // If hovered owner is not interactive, then it means that
-            // - It's not focusedOwner
-            // - It placed under focusedOwner or not exist at all
-            // In all these cases the even happened outside focused owner bounds
+        // Send Release to gestureOwner even if is not hovered or under focusedOwner
+        gestureOwner?.processPointerInput(event)
+        if (!event.isGestureInProgress) {
             val owner = hoveredOwner(event)
-            return !isInteractive(owner)
-        }
-
-        // Send Release to pressOwner even if is not hovered or under focusedOwner
-        pressOwner?.processPointerInput(event)
-        if (isOutsideFocusedOwner()) {
-            focusedOwner?.onOutsidePointerEvent?.invoke(event)
+            if (isInteractive(owner)) {
+                processHover(event, owner)
```

Comment:
> It allows to send missing event if updating during Move was blocked by pressOwner (gestureOwner)

Could you describe this use case more and write or point to the test which tests it?

Reply:
It's covered by `closeFocusablePopup` - without this `Enter` event is missed (until next move)